### PR TITLE
Comprehensive dojo tools reference

### PR DESCRIPTION
**Pull request by tinnus-napbus**
These are basically all the generators, threads and other relevant dojo commands in the `%base` desk. A handful that are either broken, deprecated, or extremely obscure have been ommitted.

### CODE REVIEW

File: content/manual/os/dojo-tools.md [hunk 1]
```diff
@@ -415,3 +497,2958 @@ None.
...
+Connect to the dojo on a remote ship (this is only possible if you have
+permission):
+
+```
+|link ~sampel-palnet %dojo
+>=
+[linked to [p=~sampel-palnet q=%dojo]]
+~sampel-palnet:dojo>
+```
+
```
Comment (sigilante) [inline content/manual/os/dojo-tools.md:786]: Are there any others in common use these days?  May be worth listing if so.  %herd

Comment (sigilante) [reply content/manual/os/dojo-tools.md:786]: Also mention that this is common with moons.

File: content/manual/os/dojo-tools.md [hunk 2]
```diff
@@ -415,3 +497,2958 @@ None.
...
+The default behavior of `+ls` is to query Clay, but [Gall][gall] [agents][agent]
+can also be queried by specifying the optional `=vane %g` argument like `+ls
+/=some-agent=/foo/bar, =vane %g`. This will perform a [scry][scry] of the form
+`.^(arch %gy /=some-agent=/foo/bar)`. Very few Gall agents implement `%y` scry
+endpoints that produce `arch` types, so this feature is almost entirely useless.
+
+#### Example
+
+```
+> +ls %/sys/vane
```
Comment (sigilante) [inline content/manual/os/dojo-tools.md:1789]: serious question:  should we de-emphasize `%` in favor of `/===`?

**Review by sigilante — APPROVED** [review]
I def. learnt some things here.

File: content/manual/os/dojo-tools.md (hunk 2 above)
Comment (tinnus-napbus) [reply content/manual/os/dojo-tools.md:1789]: I personally prefer `%`, I think `/===` is redundant. It's better to think of the use-case of `=`'s as replacing only particular fields

**Pull request merged**